packages/elements/src/components/ui/route-leg/RouteLeg.tsx

```diff
@@ -126,20 +126,20 @@ export const RouteLeg: React.FC<RouteLegProps> = ({
         />
       )}
       <div className={styles.grid}>
-        <Column alignItems={"center"} gap={0.5}>
-          <Icon icon={icon} fixedWidth size={24} />
+        <Column alignItems={"center"}>
+          <Icon icon={icon} fixedWidth size={20} />
+          <Space half />
           <Box
             flex={1}
-            border={`1px solid ${cssColor("--lhds-color-ui-300")}`}
+            border={`1px solid ${cssColor("--lhds-color-ui-900")}`}
             borderRadius={2}
           />
           <Box
-            flex={"none"}
-            border={`4px solid ${cssColor("--lhds-color-blue-100")}`}
-            background={cssColor("--lhds-color-blue-500")}
+            borderRadius={"50%"}
+            style={{ marginTop: -1 }}
```

Comment:
Dear god, why? :D

Reply:
I spent way too much time trying to keep the viewbox not crop the circle (adjusting the radius & stoke width). 

This is a simpler solution.